toolkit/tools/grapher/grapher.go

```diff
@@ -347,8 +338,7 @@ func populateGraph(graph *pkggraph.PkgGraph, repo *pkgjson.PackageRepo) (err err
 	// Rescan and add all the dependencies
 	logger.Log.Infof("Adding all dependencies from %s", *input)
 	dependenciesAdded := 0
-	for idx := range packages {
-		pkg := packages[idx]
+	for pkg := range uniquePackages {
```

Comment:
The same variable name `pkg` was used above; I wonder if it would be better to have a unique name here

Reply:
Good idea, I'll update that in a minute!